de/_posts/2019-12-07-rails-beginner-workshop.markdown

```diff
@@ -0,0 +1,10 @@
+---
+tags: beginner rails_beginner_workshop
+categories: de events
+slug: Anfänger-Workshop
```

Comment:
this optional `slug` is used instead of the file name for the `/:title` part of the permalink as defined in `_config.yml`